### PR TITLE
Log completion message flow

### PR DESCRIPTION
**Pull request by aarshkshah1992**
@raulk This is to debug the "Timed out waiting for complete message" issue.

### CODE REVIEW

File: impl/events.go [hunk 1]
```diff
@@ -244,17 +250,20 @@ func (m *manager) OnChannelCompleted(chid datatransfer.ChannelID, completeErr er
 	if completeErr == nil {
 		// If the channel was initiated by the other peer
 		if chid.Initiator != m.peerID {
+			log.Infow("received OnChannelCompleted, will send completion message to initiator", "chid", chid)
 			msg, err := m.completeMessage(chid)
 			if err != nil {
-				return nil
+				return err
```
Comment (nonsense) [inline impl/events.go:256]: Is this changed on purpose? I see everything else is log lines, apart from this.

**Review by nonsense — APPROVED** [review]
LGTM, just one comment :)

File: impl/events.go (hunk 1 above)
Comment (raulk) [reply impl/events.go:256]: This needs an explanation, as it introduces a change in behaviour.

**Review by raulk — CHANGES_REQUESTED** [review]

File: impl/events.go (hunk 1 above)
Comment (aarshkshah1992) [reply impl/events.go:256]: Hey,

The error returned from here is only logged and we take no action based on it. If there's an error while creating the complete message, we do want to know about it.

**Review by raulk — APPROVED** [review]

**Pull request merged**